docs/platform/howto/saml/add-google-idp.md

```diff
@@ -35,6 +34,7 @@ Add Google as an
     | ---------------- | --------------------------------- |
     | `Entity ID`      | `Metadata URL` from Aiven Console |
     | `ACS URL`        | `ACS URL` from Aiven Console      |
+    | `Start URL`      | <ul><li><code>https://console.aiven.io/</code> when using the Aiven Console</li> <li><code>https://console.gcp.aiven.io/</code> when using Aiven GCP Marketplace Console</li> <li><code>https://console.aws.aiven.io/</code> when using Aiven AWS Marketplace Console</li></ul>          |
```

Comment:
One new thing I forgot to mention: we can auto format table. Could you do cmd+shift+p and find `Table: format current`? 
Let me know if it doesn't work :)